lighthouse-core/runner.js

```diff
@@ -144,18 +143,20 @@ class Runner {
     return GatherRunner.run(opts.config.passes, opts).then(artifacts => {
       const flags = opts.flags;
       const shouldSave = flags.gatherMode;
-      const p = shouldSave ? Runner._saveArtifacts(artifacts): Promise.resolve();
+      const path = Runner._getArtifactsPath(flags);
+      const p = shouldSave ? Runner._saveArtifacts(artifacts, path): Promise.resolve();
```

Comment:
since we just bumped our `ecmaVersion` how about some `async/await` to kill this dumb `p` variable :D

Reply:
👍  sgtm!

Reply:
done